docker-compose.yaml

```diff
@@ -0,0 +1,10 @@
+version: "3.8"
+services:
+  postgres:
+    image: postgres:15.0-alpine3.16
+    environment:
+      - POSTGRES_USER=drand
```

Comment:
in the regression config you pass user: postgres, pass: postgres - are these actually being used?

Reply:
ahh perhaps that's docker vs unit test?

Reply:
Yes, that's correct. This is provided for development convenience rather than used in tests. I'll document this too.

Reply:
I've moved in under `test/` and added an entry in `DEVELOPMENT.md`.